Cargo.toml

```diff
@@ -28,6 +28,7 @@ libc = { version = "0.2", default-features = false }
 num-traits = { version = "0.2.15", optional=true}
 cudarc = {version = "0.9.1", optional = true }
 lazy_static = "1.4.0"
+clap = { version = "4.1.11", features = ["derive"] }
```

Comment:
Can you move it to `dev-dependencies`  this doesn't belong in core dependencies.